src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -949,11 +949,27 @@ public SamePathOrUnderConstraint SamePathOrUnder(string expected)
         /// Returns a constraint that tests whether the actual value falls 
         /// within a specified range.
         /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
+        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
         public RangeConstraint InRange(IComparable from, IComparable to)
         {
             return (RangeConstraint)this.Append(new RangeConstraint(from, to));
         }
 
+        /// <summary>
+        /// Returns a constraint that tests whether the actual value falls 
+        /// within a specified range.
+        /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
+        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
```

Comment:
[Following up](https://github.com/nunit/nunit/pull/1855#discussion_r115353617)
What about `Must be greater than or equal to the <paramref name="from"/> value.`?